specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -1632,7 +1632,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/virtualNetwork": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/networkConfig/virtualNetwork": {
```

Comment:
This may be beyond the scope of this PR, but this spec has a few operations that define `404` error status codes explicitly. AutoRest does not support these today (we have an experimental extension to handle this if you really need it). Please remove these error status codes (or apply the extension) and regenerate the SDKs

Reply:
[Here](https://github.com/Azure/autorest/blob/bc0dcde0a05606a1b102c0e08c1798fa4da4dcc8/docs/proposals/generator-specific-settings/improved-exceptions.md) is a brief explanation